pkg/daemons/executor/embed.go

```diff
@@ -47,6 +51,22 @@ func init() {
 
 func (e *Embedded) Bootstrap(ctx context.Context, nodeConfig *daemonconfig.Node, cfg cmds.Agent) error {
 	e.nodeConfig = nodeConfig
+
+	go func() {
+		//init klog v2 which is used by kube-router
+		klog.InitFlags(nil)
+		flag.Set("v", strconv.Itoa(cmds.LogConfig.VLevel))
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case <-time.After(time.Minute * 5):
+				//periodically set the verbosity level to the same value as logrus since other components can reset it
+				flag.Set("v", strconv.Itoa(cmds.LogConfig.VLevel))
```

Comment:
Every 5 minutes isn't frequent enough, and we really don't need to keep doing it after everything is started. Just doing it once a second for the first couple minutes should do the trick I think?
```suggestion
		// Ensure that the log verbosity remains set to the configured level by resetting it at 1-second intervals
		// for the first 2 minutes that K3s is starting up. This is necessary because each of the Kubernetes
		// components will initialize klog and reset the verbosity flag when they are starting.
		logCtx, _ := context.WithTimeout(ctx, time.Second * 120)
		for {
			flag.Set("v", strconv.Itoa(cmds.LogConfig.VLevel))
			select {
			case <-time.After(time.Second):
			case <-logCtx.Done():
				return
```

Reply:
Ok I didn't really understand what you meant. Thanks for your help.